tools/report-converter/codechecker_report_converter/report.py

```diff
@@ -43,10 +40,10 @@ def get_line(file_name, line_no, errors='ignore'):
                 line_no -= 1
                 if line_no == 0:
                     return line
-            return u''
+            return ''
```

Comment:
You use `plistlib.writePlist` instead of `plistlib.dumps` in this file.

:exclamation: Please check the other places too and check that we use `plistlib.load` instead of `plistlib.readPlist`